cmd/ocm/create/cluster/cmd.go

```diff
@@ -400,16 +401,16 @@ func GetDefaultClusterFlavors(connection *sdk.Connection, flavour string) (dMach
 }
 
 func getVersionOptions(connection *sdk.Connection) ([]arguments.Option, error) {
-	options, _, err := getVersionOptionsWithDefault(connection, "")
+	options, _, err := getVersionOptionsWithDefault(connection, "", nil)
```

Comment:
since the default value of the flag is an empty string, does it make sense to use the same pattern here and change the type from *string to string?

Reply:
Yes, changed to keep a string instead of *string